node/pkg/driver/node.go

```diff
@@ -664,22 +665,24 @@ func (d *NodeService) getVolumeStats(path string, volumeId string) (VolumeStatis
 	}
 
 	if isBlock {
-		mpathDevice, err := d.OsDeviceConnectivityHelper.GetMpathDevice(volumeId)
+		volumeStats, err = d.NodeUtils.GetBlockVolumeStats(volumeId)
```

Comment:
do we have a ticket to handle volume Id in the volume path for block?

Reply:
I didn't implement it because I didn't find a way to connect a volume path to volume id when the device is block, do you want me to open an investigation ticket for it?

Reply:
sure

Reply:
@matancarmeli7 could you write the ticket number here so @ArbelNathan could resolve this?

Reply:
sorry, CSI-4255